build

```diff
@@ -110,6 +110,9 @@ def compile_docker(config, target, src_dir, tgt_dir, debug=False):
     def dshell(wd, cmd):
         shell('docker run --rm -v%s:/src -v%s:/tgt -w%s %s %s' % (
             os.path.abspath(src_dir), os.path.abspath(tgt_dir), wd, dimage, cmd))
+    def dscript(wd, script):
```

Comment:
Instead of a new function, I'd prefer always passing `-v%s:/pkg` and using `os.getcwd()` as the parameter -- so you could use `dshell('/tgt/app/', '/pkg/after_compile/%s.sh')` below.